php/pantheon/checks/objectcache.php

```diff
@@ -31,7 +31,7 @@ public function run() {
     }
 
     if (empty($redis_server)) {
-      $this->alerts[] = array("message"=> "Using redis with WP-Redis object caching would speed up your backend.", "code" => 1);
+      $this->alerts[] = array("message"=> "Use redis with WP-Redis object caching to speed up your backend.[a href="https://pantheon.io/docs/articles/sites/installing-redis-on-wordpress/" target="_blank">Learn More</a>", "code" => 1);
```

Comment:
I don't think we want to assume that the message can render Markdown.  Could we use a URL shortener and just say `(see pnthn.io/sadfas)`?